src/test/java/ClassTest.java

```diff
@@ -1,80 +1,50 @@
-import com.company.WordCounter;
+import com.company.Counter;
 import org.junit.Assert;
 import org.junit.Test;
 
 import java.io.File;
 
 public class ClassTest {
 
-    @Test
-    public void ExistsAndNotEmpty() {
-
-        try {
-            File file = new File("try.txt");
-            boolean exists = file.exists();
-            long length = file.length();
-            Assert.assertTrue(exists);
-            Assert.assertNotEquals(0, length);
-        } catch (Exception e) {
-        }
-
-    }
 
     @Test
-    public void FileDoesntExist() {
-
-        try {
-            File file = new File("x.txt");
-            boolean exists = file.exists();
-            Assert.assertFalse(exists);
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
+    public void existsAndNotEmpty() {
+        Counter counter = new Counter();
```

Comment:
Jesli juz tak chcesz robic, to poprawnie jest zeby typ byl interfejsem, czyli u Ciebie WordCounter.